### PR TITLE
Add nullability extensions

### PR DESCRIPTION
**Pull request by tautropfli**

### CODE REVIEW

**Review by janhohenheim — CHANGES_REQUESTED** [review]
Add the changes to the changelog and bump the version

File: .travis.yml [hunk 1]
```diff
@@ -1,33 +1,26 @@
 dist: xenial
-language: generic
+language: csharp
```
Comment (janhohenheim) [inline .travis.yml]: revert this and the next line

**Review by janhohenheim — CHANGES_REQUESTED** [review]

**Review by janhohenheim — DISMISSED** [review]

**Comment by tautropfli** [comment]
Closing in favour of #10

**Review by janhohenheim — COMMENTED** [review]
Per discussion: We actually use `foo is { }` now.

**Pull request closed without merging**